nipype/algorithms/misc.py

```diff
@@ -1520,3 +1418,14 @@ def __init__(self, **inputs):
         warnings.warn(("This interface has been deprecated since 0.10.0,"
                        " please use nipype.algorithms.metrics.FuzzyOverlap"),
                       DeprecationWarning)
+
+class TSNR(confounds.TSNR):
+    """
+    .. deprecated:: 0.12.1
+       Use :py:class:`nipype.algorithms.confounds.TSNR` instead
+    """
+    def __init__(self, **inputs):
+        super(confounds.TSNR, self).__init__(**inputs)
+        warnings.warn(("This interface has been deprecated since 0.12.0,"
```

Comment:
Could we say moved instead of deprecated? "deprecated" suggest that functionality has changed